app/src/main/java/com/kickstarter/models/UserPrivacy.kt

```diff
@@ -0,0 +1,11 @@
+package com.kickstarter.models
+
+data class UserPrivacy(
```

Comment:
Don't take the User query response and directly pass values from it around app. Create a new data class that the User query response must get transformed to before passing it around app.